akka-projection-alpakka-kafka/src/main/scala/akka/projection/scaladsl/kafka/KafkaProjection.scala

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.scaladsl.kafka
+
+import akka.kafka.CommitterSettings
+import akka.kafka.ConsumerMessage.{ CommittableMessage, CommittableOffset }
+import akka.kafka.scaladsl.Consumer.{ Control, DrainingControl }
+import akka.kafka.scaladsl.{ Committer, Consumer }
+import akka.projection.scaladsl._
+import akka.stream.Materializer
+import akka.stream.scaladsl.{ Keep, Sink, Source }
+import akka.{ Done, NotUsed }
+import org.apache.kafka.clients.consumer.ConsumerRecord
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+object KafkaProjection {
+
+  def apply[K, V](
+      src: Source[ConsumerRecord[K, V], Control],
+      handler: EventHandler[V, Future[Done]]): Projection[ConsumerRecord[K, V], V, NotUsed, Future[Done]] =
+    new ConsumerRecordProjection(src, handler)
+
+  def apply[K, V, R](
+      sourceProvider: SourceProvider[Long, ConsumerRecord[K, V], Control],
```

Comment:
this is still being used, but maybe we should just a function: `Option[Offset] => Source[..]`. 
So users can tweak it better.